circle.yml

```diff
@@ -14,13 +14,14 @@ dependencies:
     - npm install saucelabs@0.1.1 # allow sending Watai results to SauceLabs
     - wget https://saucelabs.com/downloads/sc-latest-linux.tar.gz
     - tar -xzf sc-latest-linux.tar.gz
-    - sudo pip install --upgrade pip
-    - sudo pip install gunicorn
-    - npm run install-openfisca
+    - virtualenv --python=python2.7 ~/virtualenv
+    - source ~/virtualenv/bin/activate && pip install --upgrade pip
+    - source ~/virtualenv/bin/activate && pip install gunicorn
+    - source ~/virtualenv/bin/activate && npm run install-openfisca
```

Comment:
It does only when it detects a Python project.
I tried with a single `source` (cf. https://github.com/sgmap/mes-aides-ui/pull/567/commits/5c9a74db27bf9031c4fecd2b14286f418b850c02) but [it failed](https://circleci.com/gh/sgmap/mes-aides-ui/1737).